docker-compose.yml.example

```diff
@@ -11,13 +11,12 @@ services:
       - /etc/localtime:/etc/localtime:ro
     ports:
       - "9042:9042"
-# fix after putting curl or wget into image.
-#    healthcheck:
-#      test: curl --fail http://localhost:9042/status || exit 1
-#      interval: 60s
-#      timeout: 30s
-#      retries: 3
-#      start_period: 30s
+    healthcheck:
+      test: ["CMD", "wget", "--no-verbose", "--tries=1", "--spider", "http://localhost:9042/status"]
```

Comment:
Hm, somehow we need to ensure this exits with 1 on fail tho (and not another code). Does the method like the commented out curl command not work ?

Reply:
Tried to trigger it with just another port to see, and should behave as expected. At least the best test I could come up with.

{
	"Start": "2024-03-15T15:27:11.521671334Z",
	"End": "2024-03-15T15:27:11.562590376Z",
	"ExitCode": 1,
	"Output": "Connecting to localhost:9043 (127.0.0.1:9043)\nwget: can't connect to remote host (127.0.0.1): Connection refused\n"
}

Reply:
Yeah, works for that. And I think this is prob good enough. But just for curiousity, can you see what the exit code is if you plug in a wrong IP address (so it imitates a timeout)?  (Pick a local IP that is not being used by anything)

Reply:
Calling an older version of Fletchling without status endpoint
{
"Start": "2024-03-15T16:18:15.701784382Z",
"End": "2024-03-15T16:18:15.759087799Z",
"ExitCode": 1,
"Output": "Connecting to 192.168.1.15:9042 (192.168.1.15:9042)\nwget: server returned error: HTTP/1.1 404 Not Found\n"

Calling an unused local ip. Still seems to set the container as unhealthy. So not sure how much it matters.
{
"Start": "2024-03-15T16:30:04.221639585Z",
"End": "2024-03-15T16:30:34.261224919Z",
"ExitCode": -1,
"Output": "Health check exceeded timeout (30s): Connecting to 192.168.1.16:9042 (192.168.1.16:9042)\n"
}

Reply:
Ok, if it sets the container unhealthy with exit code of -1 there too, then all good. 

Thanks for testing those.